eng/pipelines/libraries/build-test-job.yml

```diff
@@ -56,7 +56,7 @@ jobs:
             cleanUnpackFolder: false
 
         - script: $(_buildScript)
-                -subset pretest-libtests
+                -subset libs.pretest+libs.libtests
```

Comment:
`libs.libtests` looks weird, would it be better as `libs.tests`? we already have the prefix before the dot.

Reply:
Oh I see this is actually a mistake here in the yml.

Reply:
lol, that explains the failures.